mmdet3d/ops/paconv/paconv.py

```diff
@@ -72,7 +76,12 @@ def __init__(self,
 
     def init_weights(self):
         """Initialize weights of shared MLP layers."""
+        if not self.use_xavier_init:
+            return
+        # the official code applies xavier_init to all the Conv layers
         # refer to https://github.com/CVMI-Lab/PAConv/blob/main/scene_seg/model/pointnet2/paconv.py#L105  # noqa
+        # however in our experiments, we did not find much difference in
+        # applying the xavier initialization or not
```

Comment:
if so, we can ommit this mode, do not support that but leave a comment

Reply:
OK